### ISSUE TITLE
Response validation

### ISSUE THREAD

**Issue by alongreyber** [TranslatorSRI/reasoner-pydantic#7]
This code does not throw a validation error:

```python
Response.parse_obj(
    {"message" : []}
)
```

Based on my understanding this should probably throw an error because message is a dict. Let me know if I am missing something.

**Comment by patrickkwang** [comment]
You're right, this is odd. If "message" is a non-empty list, it seems to fail validation as expected. If the value is the empty string validation passes, but non-empty strings fail. `False` and 0 also fail validation, so it's not just passing falsy values.

**Comment by patrickkwang** [comment]
This is a known pydantic bug: https://github.com/samuelcolvin/pydantic/issues/1268
